lib/logstash/inputs/gelf.rb

```diff
@@ -33,6 +33,15 @@ class LogStash::Inputs::Gelf < LogStash::Inputs::Base
   # ports) may require root to use.
   config :port, :validate => :number, :default => 12201
 
+  # The GELF protocol (TCP or UDP).
+  config :protocol, :validate => [ "UDP", "TCP" ], :default => "UDP"
+
+  # Whether to capture the hostname or numeric address of the incoming connection
+  # in protocol tcp.
+  #
+  # Defaults to hostname
+  config :use_numeric_client_addr, :validate => :boolean, :default => false
```

Comment:
I would prefer not abbreviating `address`.

I wonder if this could be a better name. Some questions for thinking about this:

* Do any other plugins have a similar option? 
* In what situations would a user want to set this to true?
* This setting seems, when `true`, to describe not-acting (meaning, to *not* do a dns lookup). Can we think of a positive-action name for this setting instead of a negative-action?